src/xrGame/WeaponBinoculars.cpp

```diff
@@ -124,7 +124,7 @@ void CWeaponBinoculars::ZoomDec()
     float delta, min_zoom_factor;
     GetZoomData(m_zoom_params.m_fScopeZoomFactor, delta, min_zoom_factor);
 
-    float f = GetZoomFactor() + delta;
+    float f = GetZoomFactor() - delta;
```

Comment:
This is gameplay change and it could be done intentionally

Reply:
The final result that the end-user sees is unaffected, so this does not change gameplay at all. Just makes more logical sense in code (imo).

If i revert the changes in `WeaponBinoculars.cpp`, then gameplay behavior would be changed.